README.md

```diff
@@ -12,34 +12,19 @@ MIT Licensed.
 Usage
 -----
 
-Available flags for `unifi_exporter` include:
-
 ```
 $ ./unifi_exporter -h
 Usage of ./unifi_exporter:
-  -telemetry.addr string
-        host:port for UniFi exporter (default ":9130")
-  -telemetry.path string
-        URL path for surfacing collected metrics (default "/metrics")
-  -unifi.addr string
-        address of UniFi Controller API
-  -unifi.insecure
-        [optional] do not verify TLS certificate for UniFi Controller API (warning: please use carefully)
-  -unifi.password string
-        password for authentication against UniFi Controller API
-  -unifi.site string
-        [optional] description of site to collect metrics for using UniFi Controller API; if none specified, all sites will be scraped
-  -unifi.timeout duration
-        [optional] timeout for UniFi Controller API requests (default 5s)
-  -unifi.username string
-        username for authentication against UniFi Controller API
+  -config.file.path string
```

Comment:
Do we need a nested option here? `config` or `config.path` would be sufficient imo.

Reply:
Probably not. Will fix tomorrow.

Reply:
This should be in sync with the actual flag name from the code -> `config.file`